test/CMakeLists.txt

```
@@ -11,6 +11,7 @@ if(GTest_FOUND)
         io_read.cpp
         io_wr.cpp
         completeness.cpp
+		correctness.cpp
```

```suggestion
        correctness.cpp
```
This file isn't processed by formatters, and the indentation was incorrect (changed tabs to spaces)